__tests__/util/geocoder.js

```diff
@@ -0,0 +1,106 @@
+import nock from 'nock'
+
+import getGeocoder from '../../lib/util/geocoder'
+
+function mockResponsePath (geocoder, file) {
+  return `__tests__/test-utils/fixtures/geocoding/${geocoder}/${file}`
+}
+
+describe('geocoder', () => {
+  const geocoders = [
+    {
+      apiKey: 'dummy-mapzen-key',
+      baseUrl: 'https://ws-st.trimet.org/pelias/v1',
+      type: 'PELIAS'
+    }, {
+      type: 'ARCGIS'
```

Comment:
Why is no baseUrl defined for ARCGIS? Isn't this required by the `makeGeocoder` func?

Reply:
Oh, never mind. I guess the default base URL is set in the arcgis geocoder library.